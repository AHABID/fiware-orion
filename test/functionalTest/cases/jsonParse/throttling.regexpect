1: +++++++++++++++++++++++++++++
HTTP/1.1 200 OK
Content-Length: 137
Content-Type: application/json
Date: REGEX(.*)

{
    "subscribeResponse": {
        "duration": "PT24H", 
        "subscriptionId": "REGEX([0-9a-f]{24})", 
        "throttling": "PT5H"
    }
}
2: +++++++++++++++++++++++++++++
