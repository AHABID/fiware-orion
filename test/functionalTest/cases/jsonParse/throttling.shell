echo "1: +++++++++++++++++++++++++++++"
url="/ngsi10/subscribeContext"
payload='{
  "entities" : [
    {
      "type" : "Room",
      "isPattern" : "false",
      "id" : "OfficeRoom"
    }
  ],
  "reference" : "http://localhost:${LISTENER_PORT}/notify",
  "throttling" : "PT5H"
}'
curlJson ${url} "${payload}"

echo "2: +++++++++++++++++++++++++++++"
