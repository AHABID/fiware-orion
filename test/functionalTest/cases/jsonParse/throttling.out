1: +++++++++++++++++++++++++++++
HTTP/1.1 200 OK
Content-Length: 137
Content-Type: application/json
Date: Mon, 02 Jun 2014 10:01:38 GMT

{
    "subscribeResponse": {
        "duration": "PT24H", 
        "subscriptionId": "538c4b82c0fc487480492f7e", 
        "throttling": "PT5H"
    }
}
2: +++++++++++++++++++++++++++++
